### DISCUSSION TITLE
Element implicitly has an 'any' type because type 'typeof globalThis' has no index signature.

### DISCUSSION THREAD

**Discussion by CompeyDev** [prisma/prisma#14468] [Q&A] [answered]
Ran into a error similar to #10037 when trying to use Prisma + NextJS from a PlanetScale article. 
![image](https://user-images.githubusercontent.com/74418041/180639259-88f8806a-01f7-4974-b06f-329f93e2bdfb.png)

If I change it to the code in the solution, then my model shows up as non existent in my other files:
![image](https://user-images.githubusercontent.com/74418041/180639283-07c8da17-d12a-4a1f-ba9b-5d9ef7b88536.png)

Edit: Tried restarting TypeScript server as well.

**Comment by austincrim** [comment, marked as answer]
Hey @CompeyDev 👋,

Have you tried the workaround specified in the linked issue?

```ts
import { PrismaClient } from '@prisma/client';

let prisma: PrismaClient;

if (process.env.NODE_ENV === "production") {
  prisma = new PrismaClient();
} else {
  let globalWithPrisma = global as typeof globalThis & {
    prisma: PrismaClient;
  };
  if (!globalWithPrisma.prisma) {
    globalWithPrisma.prisma = new PrismaClient();
  }
  prisma = globalWithPrisma.prisma;
}

export default prisma;
```

**Comment by CompeyDev** [comment]
Yeah, tried that too. Turns out (for some reason), models are now lowercase in the generated client? Is this expected or..?

**Reply by austincrim** [reply]
Model names are always lowercased in the generated client. There is no way to configure this right now, but we [have an outstanding issue](https://github.com/prisma/prisma/issues/1184) to allow more introspection configuration, including field name mappings.

Feel free to add your use case and +1 in that issue or open a new [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you feel that your situation isn't totally related.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
